Build error

```
    Compiling process_path v0.1.4
error[E0433]: failed to resolve: use of undeclared crate or module `os`
  --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/process_path-0.1.4/src/lib.rs:58:5
   |
58 |     os::get_executable_path()
   |     ^^ use of undeclared crate or module `os`

error[E0433]: failed to resolve: use of undeclared crate or module `nix`
  --> /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/process_path-0.1.4/src/lib.rs:71:9
   |
71 |         nix::get_dylib_path()
   |         ^^^ use of undeclared crate or module `nix`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `process_path` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,5 +13,5 @@ keywords = ["current", "process", "executable", "dylib", "dll"]
 [target.'cfg(windows)'.dependencies]
 winapi = { version = "0.3.9", features = ["errhandlingapi", "libloaderapi", "minwindef", "winerror" ] }
 
-[target.'cfg(any(target_os="linux", target_os="freebsd", target_os="dragonfly", target_os="netbsd", target_os="macos", target_os="illumos"))'.dependencies]
+[target.'cfg(any(target_os="linux", target_os="freebsd", target_os="dragonfly", target_os="netbsd", target_os="macos", target_os="illumos", target_os="android"))'.dependencies]
 libc = "0.2.81"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -21,11 +21,12 @@
 //! * macOS
 //! * Windows
 //! * illumos
+//! * Android
 use std::path::PathBuf;
 
-#[cfg(any(target_os = "linux", target_os = "illumos"))]
+#[cfg(any(target_os = "linux", target_os = "illumos", target_os = "android"))]
 mod linux;
-#[cfg(any(target_os = "linux", target_os = "illumos"))]
+#[cfg(any(target_os = "linux", target_os = "illumos", target_os = "android"))]
 use linux as os;
 
 #[cfg(target_os = "macos")]
@@ -44,7 +45,8 @@ use bsd as os;
     target_os = "dragonfly",
     target_os = "netbsd",
     target_os = "macos",
-    target_os = "illumos"
+    target_os = "illumos",
+    target_os = "android"
 ))]
 mod nix;
 

